cmd/keeper/cmd/keeper.go

```diff
@@ -443,6 +445,8 @@ type PostgresKeeper struct {
 	sleepInterval  time.Duration
 	requestTimeout time.Duration
 
+	disableAlterSystem bool
```

Comment:
@sgotti pointed out in a separate PR that flags are usually much more intuitive when they're positive, ie. `enable-alter-system`, with a default `false`.

For consistency, could we change disable->enable please?